cluster/common.sh

```diff
@@ -486,6 +483,35 @@ function verify-kube-binaries() {
   "${get_binaries_script}"
 }
 
+# Check whether required release artifacts exist, prompting to download
+# if missing.
+# If KUBERNETES_SKIP_CONFIRM is set to y, we'll automatically download binaries
+# without prompting.
+function verify-release-tars() {
+  local missing_tars=false
+  if ! $(find-release-tars); then
+    missing_tars=true
+  fi
+
+  if ! "${missing_tars}"; then
+    return
+  fi
+
+  get_binaries_script="${KUBE_ROOT}/cluster/get-kube-binaries.sh"
```

Comment:
rather than copy-pasting this from `verify-kube-binaries`, maybe refactor this block into a separate `download-release-artifacts` function?

Reply:
done